src/DebugBar/DebugBar.php

```diff
@@ -83,21 +89,30 @@ public function render()
     {
         $renderer = $this->bar->getJavascriptRenderer();
         $renderer->setIncludeVendors('css');
-        $asset_hash = \md5(\json_encode($renderer->getAssets()));
-        $base_url   = '/assets/compiled/debug-bar';
-        $base_path  = DOCROOT.$base_url;
-        if ( ! \is_dir($base_path)) {
-            \mkdir($base_path, 0777, TRUE);
-        }
-        if ( ! \file_exists("$base_path/debug-$asset_hash.css")) {
-            $renderer->dumpCssAssets("$base_path/debug-$asset_hash.css");
-        }
-        if ( ! \file_exists("$base_path/debug-$asset_hash.js")) {
-            $renderer->dumpJsAssets("$base_path/debug-$asset_hash.js");
+
+        $debugbar_root = '/dev-assets/debugbar/';
+        $hash          = md5(json_encode($renderer->getAssets()));
+        $asset_url     = $debugbar_root.$hash;
+        $asset_path    = DOCROOT.$asset_url;
+        if ( ! is_dir($asset_path)) {
+            mkdir($asset_path, 0777, TRUE);
+            $renderer->dumpCssAssets("$asset_path/debugbar.css");
+            $renderer->dumpJsAssets("$asset_path/debugbar.js");
+
+            $font_path = DOCROOT.$debugbar_root.'fonts';
+            if ( ! is_dir($font_path)) {
+                mkdir($font_path, 0777, TRUE);
+                $src = APPPATH.'../vendor/maximebf/debugbar/src/DebugBar/Resources/vendor/font-awesome/fonts';
+                foreach (scandir($src) as $file) {
```

Comment:
Could use `glob` to save the extra check for `.` / `..`

Reply:
Ahh thanks for that, I knew there was a function out there and couldn't remember which one at all, scandir and readdir were the most obvious when I was searching.